pkg/monitor/monitor/util.go

```diff
@@ -319,7 +331,118 @@ chmod 777 /data/prometheus /data/grafana
 	return container
 }
 
-func getMonitorPrometheusContainer(monitor *v1alpha1.TidbMonitor, tc *v1alpha1.TidbCluster) core.Container {
+func getMonitorDMInitContainer(monitor *v1alpha1.TidbMonitor, dc *v1alpha1.DMCluster) core.Container {
```

Comment:
There is much duplicate code with `getMonitorInitContainer`, could you please define some functions and optimize them?

Reply:
fixed